client/state/selectors/get-active-discount.js

```diff
@@ -3,7 +3,7 @@
 /**
  * External dependencies
  */
-import { isPlainObject } from 'lodash';
+import _ from 'lodash';
```

Comment:
Any reason for importing all of `lodash`? Or did I miss the preferred way for it?

Reply:
Cool